pkg/kubernetes/app.js

```diff
@@ -162,15 +162,17 @@ define([
                             }
                         };
 
-                        scope.select = function select(id, value) {
+                        scope.select = function select(id, ev) {
+                            var value;
+                            if (ev && $(ev.target).parents().andSelf().filter(".btn, li").length > 0)
```

Comment:
[andSelf() is deprecated](https://api.jquery.com/andself/)
such a long chain also might deserve a comment